### PR TITLE
Add Rayman 3D

### PR DESCRIPTION
**Pull request by greggameplayer**
Adds Rayman 3D.

*Automatically generated by PR builder*

### CODE REVIEW

File: games/rayman-3d/game.dat [hunk 1]
```diff
@@ -0,0 +1,19 @@
+title = "Rayman 3D"
+description = "Rayman 3D is a direct port of the Sega Dreamcast version of Rayman 2: The Great Escape. The game is in full 3D and has enhanced graphics and widescreen display."
```
Comment (Leo626) [inline games/rayman-3d/game.dat]: Should mention the developer and publisher.

> Rayman 3D is an action-adventure video game developed and published by Ubisoft. It is a direct port of the Sega Dreamcast version of Rayman 2: The Great Escape. The game is in full 3D and has enhanced graphics and widescreen display.

File: games/rayman-3d/game.dat [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+title = "0004000000033900"
+region = "eur"
+release_date = "2011-03-25"
+
+[[testcases]]
+title = "0004000000033900"
+compatibility = "1"
+date = "2017-11-30"
+version = "HEAD-b2a9904"
+cpu = "AMD FX 6300"
```
Comment (Leo626) [inline games/rayman-3d/game.dat]: `FX-6300` missing `-`

File: games/rayman-3d/game.dat [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+needs_shared_font = true
+
+[[releases]]
+title = "0004000000033900"
+region = "eur"
+release_date = "2011-03-25"
+
+[[testcases]]
+title = "0004000000033900"
+compatibility = "1"
```
Comment (Leo626) [inline games/rayman-3d/game.dat]: The wiki mentions no graphical or audio issues and is perfectly playable. Unless there's anything such as crashing or freezing that prevents progress in the game, the rating should be `0`.

**Comment by greggameplayer** [comment]
Ok I've done all things you've suggest to me.

**Comment by greggameplayer** [comment]
@Leo626 or another person, can you merge my branch ?

**Comment by Leo626** [comment]
@greggameplayer It's normally two approvals before it gets merged.

**Comment by greggameplayer** [comment]
ok

**Comment by greggameplayer** [comment]
@BreadFish64 can you approve the changes please ?

**Comment by greggameplayer** [comment]
@Hexagon12 , @TheKoopaKingdom or anyone else can you approve my changes please ?

**Comment by greggameplayer** [comment]
so now @Leo626 , @BreadFish64 or anyone else can you merge my branch please ?

**Comment by greggameplayer** [comment]
just another question how can I get the write access ? because I have a lot of games reports to push and it's really heavy to wait 2 or 3 days for branch merging and I want to help you to suggest the other game report.

**Comment by BreadFish64** [comment]
I don't even have write access

**Comment by greggameplayer** [comment]
but @Leo626 has the write access isn't it ?

**Comment by Leo626** [comment]
@greggameplayer If you're familiar with git you can add multiple games into one pull request as long as there aren't too many. The PR builder in the website only allows for one game at a time.

**Pull request merged**